Readme.md

```diff
@@ -36,118 +36,135 @@ Please read the [FAQ](https://dolphin-emu.org/docs/faq/) before using Dolphin.
-
-To install to your system.
-
-1. `mkdir build`
-2. `cd build`
-3. `cmake ..`
-4. `make -j $(nproc)`
-5. `sudo make install`
-
-### Linux Local Build Steps:
-
-Useful for development as root access is not required.
-
-1. `mkdir Build`
-2. `cd Build`
-3. `cmake .. -DLINUX_LOCAL_DEV=true`
-4. `make -j $(nproc)`
-5. `ln -s ../../Data/Sys Binaries/`
-
-### Linux Portable Build Steps:
-
-Can be stored on external storage and used on different Linux systems.
-Or useful for having multiple distinct Dolphin setups for testing/development/TAS.
-
-1. `mkdir Build`
-2. `cd Build`
-3. `cmake .. -DLINUX_LOCAL_DEV=true`
-4. `make -j $(nproc)`
-5. `cp -r ../Data/Sys/ Binaries/`
-6. `touch Binaries/portable.txt`
-
-## Building for Android
-
-These instructions assume familiarity with Android development. If you do not have an
-Android dev environment set up, see [AndroidSetup.md](AndroidSetup.md).
-
-Make sure to pull submodules before building:
+#### CMake local build
+Pass `-DLINUX_LOCAL_DEV=true` to CMake when creating the build directory, and
+put a symlink to Dolphin's assets in the same directory as the binaries:
 ```sh
-git submodule update --init --recursive
+cmake -DLINUX_LOCAL_DEV=true -G Ninja -B build &&
+  cmake --build build &&
+  ln -s Data/Sys build/Binaries/
 ```
+Useful for development, as root access is not required.
 
-If using Android Studio, import the Gradle project located in `./Source/Android`.
-
-Android apps are compiled using a build system called Gradle. Dolphin's native component,
-however, is compiled using CMake. The Gradle script will attempt to run a CMake build
-automatically while building the Java code.
-
-## Uninstalling
-
-On Windows, simply remove the extracted directory, unless it was installed with the NSIS installer,
-in which case you can uninstall Dolphin like any other Windows application.
+#### CMake portable build
+Pass `-DLINUX_LOCAL_DEV=true` to CMake when creating the build directory,
+copypaste Dolphin's assets into the same directory as the binaries, and create a
+`portable.txt` file in the same directory as the binaries:
+```sh
+cmake -DLINUX_LOCAL_DEV=true -G Ninja -B build &&
+  cmake --build build &&
+  cp -r Data/Sys build/Binaries/ &&
+  touch build/Binaries/portable.txt
+```
+`build/Binaries` can be stored on external storage and used on different
+systems. Also useful for having multiple distinct Dolphin setups for testing,
+development, or TAS.
+
+#### macOS universal binaries
+Doing this is more complex as it requires installation of library dependencies
+for both x64 and ARM (or universal library equivalents) and may require
+specifying additional arguments to point to relevant library locations.
+Run the following command for more details:
+```sh
+BuildMacOSUniversalBinary.py --help
+```
+To build universal binaries supporting both x64 and ARM in the same application
+bundle, run the following command:
+```sh
+mkdir build &&
+  (cd build && python ../BuildMacOSUniversalBinary.py)
```

Comment:
Any reason why you combined those two into a single line, rather than just making it 3 total?
```suggestion
mkdir build &&
  cd build &&
  python ../BuildMacOSUniversalBinary.py
```

Reply:
With the subshell (inside `()`), it doesn't change directory into the build directory outside the subshell. It just felt like it made sense to keep the whole subshell in one line. Tell me if I should still change it.

Reply:
Interresting, I wasn't aware this was a thing. Probably makes sense to keep it that way then 👍